### PR TITLE
lookout chart

### PR DESCRIPTION
**Pull request by rporres**
Signed-off-by: Rafael Porres Molina <rafa@sourced.tech>

### CODE REVIEW

File: lookout/values.yaml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  lookout: 1
+  dummyAnalyzer: 1
+
+image:
+  lookout:
+    repository: srcd/lookout
+    #tag: latest
+    pullPolicy: IfNotPresent
+  dummyAnalyzer:
+    repository: smacker/dummy-analyzer
```
Comment (bzz) [inline lookout/values.yaml]: A noted in https://github.com/src-d/issues-infrastructure/issues/213#issuecomment-413472338, let's change this `srcd/lookout-dummy-analyzer`

Comment (rporres) [reply lookout/values.yaml]: fixed in c495d7a

File: lookout/templates/dummy-analyzer-deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        app: dummy-analyzer
+        release: {{ .Release.Name }}
+    spec:
+      containers:
+        - name: dummy-analyzer
+          image: "{{ .Values.image.dummyAnalyzer.repository }}:{{ .Values.image.dummyAnalyzer.tag }}"
+          imagePullPolicy: {{ .Values.image.dummyAnalyzer.pullPolicy }}
+          args:
+            - serve
+            - {{ .Values.app.lookout.repository }}
```
Comment (smacker) [inline lookout/templates/dummy-analyzer-deployment.yaml]: analyzer doesn't need repository argument. It can work with any repo sent by lookout server.

Comment (rporres) [reply lookout/templates/dummy-analyzer-deployment.yaml]: fixed in a3951ba

**Comment by bzz** [comment]
Looks great, thank you @rporres ! 

May be let's keep it open while CD is being configured, just in case of any possible modifications.

**Comment by rporres** [comment]
> May be let's keep it open while CD is being configured, just in case of any possible modifications.

I don't agree with that. If you're happy with the chart as it is now, let's merge it. If we want further modifications when new things are needed, we can create a new one

**Pull request merged**